t/web/house.rb

```diff
@@ -19,4 +19,11 @@
       subject.css('.avatar-unit p').last.text.must_equal '1981-01-06 - 1983-01-03'
     end
   end
+
+  describe 'HTML validation' do
+    it 'has no errors in the house page' do
+      skip if `which tidy`.empty?
```

Comment:
I don't think we want to duplicate this in every test like this. I'd suggest pulling this up into `last_response_must_be_valid`